debian/ellis.init.d

```diff
@@ -124,26 +92,18 @@ do_start()
 #
 do_stop()
 {
-	# Return
-	#   0 if daemon has been stopped
-	#   1 if daemon was already stopped
-	#   2 if daemon could not be stopped
-	#   other if a failure occurred
-
-        # Stopping Ellis terminates all processes that use its version of
-        # python, including any of its tools. Wait for the tools to complete,
-        # otherwise we can break automated installs.
-        wait_for_tools || return 2
```

Comment:
Per the comment: "Stopping Ellis terminates all processes that use its version of python" - this was because we previously ran start-stop-daemon with just the executable as a match. Now that we're specifically targeting it by pidfile, we don't need this.

I'm happy to create a cw_aio node in EC2 to verify this.

(The original issue was https://github.com/Metaswitch/ellis/pull/106, for reference.)